src/test/java/se/sundsvall/installation/service/InstallationServiceTest.java

```diff
@@ -1,4 +1,35 @@
 package se.sundsvall.installation.service;
 
-public class InstallationServiceTest {
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static se.sundsvall.installation.TestUtil.createSearchParameters;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import se.sundsvall.installation.integration.datawarehousereader.DataWarehouseReaderClient;
+
+import generated.se.sundsvall.datawarehousereader.InstallationParameters;
+
+@ExtendWith(MockitoExtension.class)
+class InstallationServiceTest {
+
+	@Mock
+	private DataWarehouseReaderClient dataWarehouseReaderClient;
+
+	@InjectMocks
+	private InstallationService installationService;
+
+	@Test
+	void getInstallations() {
+		final var searchParameters = createSearchParameters();
+
+		installationService.getInstallations(searchParameters);
+
+		verify(dataWarehouseReaderClient).getInstallationDetails(any(InstallationParameters.class));
```

Comment:
Use captor here to verify that correct values are passed to the dataWarehouseReaderClient